nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/PutEmail.java

```diff
@@ -79,10 +82,17 @@
 @Tags({"email", "put", "notify", "smtp"})
 @InputRequirement(Requirement.INPUT_REQUIRED)
 @CapabilityDescription("Sends an e-mail to configured recipients for each incoming FlowFile")
```

Comment:
The annotation `@SupportsSensitiveDynamicProperties` should be added to allow optional property encryption for properties such as Proxy passwords.

Reply:
That makes sense, adding it!